internal/venice-common/src/main/proto/VeniceEchoService.proto

```diff
@@ -0,0 +1,29 @@
+syntax = 'proto3';
+package com.linkedin.venice.protocols;
+
+import "google/rpc/status.proto";
+import "google/rpc/error_details.proto";
+import "google/protobuf/descriptor.proto";
+
+import "VeniceStore.proto";
+
+option java_multiple_files = true;
+
+// Define a custom option for method types
+extend google.protobuf.MethodOptions {
+  string methodType = 50001; // Assign a unique field number
+}
+
+service VeniceEchoService {
```

Comment:
What is this for? Just some kind of hello world? Or a health check?

Reply:
Ya, this is hello world kind of service that is only used for testing of `VeniceGrpcServer` class

Reply:
Replaced this with controller service. `VeniceEchoService` will be removed since we no longer need it.